packages/lesmis/src/views/DashboardView.js

```diff
@@ -480,14 +480,27 @@ const VitalsView = React.memo(({ vitals }) => {
   }
 });
 
+// Gets the best default dashboard possible, and check if the selected dashboard is valid
+const useDefaultDashboardTab = (selectedDashboard = null, options) => {
+  if (!options) {
+    return null;
+  }
+
+  if (selectedDashboard && options.find(option => option.value === selectedDashboard)) {
+    return selectedDashboard;
+  }
+
+  return options[0].value;
```

Comment:
There's a chance here that `options` is an empty array

Reply:
Yes good point. I've added `options.length === 0` to the early return condition